### PR TITLE
[tests-only] [full-ci] Test to check move activity of file

### PR DESCRIPTION
**Pull request by Salipa-Gurung**
## Description
Added test to check activity for move file event.

## Related Issue
- Part of https://github.com/owncloud/ocis/issues/9543
- https://github.com/owncloud/ocis/issues/9712

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/apiGraph/activities.feature [hunk 1]
```diff
@@ -9,7 +9,7 @@ Feature: check activities
   @issue-9712
   Scenario: check activities after uploading a file
     Given user "Alice" has uploaded file with content "ownCloud test text file 0" to "/textfile0.txt"
-    When user "Alice" checks the activities for file "textfile0.txt" in space "Personal" using the Graph API
+    When user "Alice" lists the activities for file "textfile0.txt" of space "Personal" using the Graph API
```
Comment (saw-jan) [inline tests/acceptance/features/apiGraph/activities.feature:12]: :+1:

File: tests/acceptance/features/apiGraph/activities.feature [hunk 2]
```diff
@@ -367,3 +367,172 @@ Feature: check activities
         }
       }
       """
+
+  @issue-9712 @issue-9744
+  Scenario: check move activity for a file
```
Comment (saw-jan) [inline tests/acceptance/features/apiGraph/activities.feature:372]: the issue is related to MOVE using file-id only
```suggestion
  @issue-9712
  Scenario: check move activity for a file
```

File: tests/acceptance/features/apiGraph/activities.feature [hunk 3]
```diff
@@ -367,3 +367,172 @@ Feature: check activities
...
+    Given user "Alice" has uploaded file with content "ownCloud test text file 0" to "/textfile0.txt"
+    And user "Alice" has created folder "/New Folder"
+    And user "Alice" has moved file "textfile0.txt" to "New Folder/textfile0.txt"
+    When user "Alice" lists the activities of space "Personal" using the Graph API
```
Comment (saw-jan) [inline tests/acceptance/features/apiGraph/activities.feature:376]: check the activity of a file only
```suggestion
    When user "Alice" lists the activities for file "New Folder/textfile0.txt" of space "Personal" using the Graph API
```

File: tests/acceptance/features/apiGraph/activities.feature [hunk 4]
```diff
@@ -367,3 +367,172 @@ Feature: check activities
...
+                              "required": ["id","name"],
+                              "properties": {
+                                "name": {
+                                  "const": "textfile0.txt"
+                                }
+                              }
+                            }
+                          }
+                        }
+                      }
```
Comment (saw-jan) [inline tests/acceptance/features/apiGraph/activities.feature:417]: `message` should be enough for old activities
```suggestion
```

Comment (Salipa-Gurung) [reply tests/acceptance/features/apiGraph/activities.feature:417]: wouldn't that be confusing to know which resource it is for?

Comment (saw-jan) [reply tests/acceptance/features/apiGraph/activities.feature:417]: for a single item, we should get the activities of that item only as noted here https://github.com/owncloud/ocis/pull/9731#discussion_r1714759795
That we won't have to deal with multiple items. but the point is true for folder activities :+1:

**Review by saw-jan — APPROVED** [review]
LGTM :+1:

**Review by ScharfViktor — APPROVED** [review]
LGtM

**Pull request merged**